tests/foreman/ui/test_computeresource_rhev.py

```diff
@@ -200,12 +344,38 @@ def test_negative_add_image_rhev_with_invalid_name(self):
         5. Select the desired template to create the image from and submit.
 
         @Assert: The image should not be added to the CR
-
-        @Caseautomation: notautomated
         """
+        parameter_list = [
+                        ['URL', self.rhev_url, 'field'],
+                        ['Username', self.rhev_username, 'field'],
+                        ['Password', self.rhev_password, 'field'],
+                        ['Datacenter', self.rhev_datacenter, 'special select']
+                        ]
+        name = gen_string('alpha')
+        with Session(self.browser) as session:
+            for img_name in invalid_names_list():
+                with self.subTest(name):
+                    make_resource(
+                                  session,
+                                  name=name,
+                                  provider_type=FOREMAN_PROVIDERS['rhev'],
+                                  parameter_list=parameter_list
+                                  )
+                parameter_list1 = [
+                           ['Name', img_name],
+                           ['Operatingsystem', self.rhev_img_os],
+                           ['Architecture', self.rhev_img_arch],
+                           ['Username', self.rhev_img_user],
+                           ['Password', self.rhev_img_pass],
+                           ['uuid', self.rhev_img_name],
+                           ]
```

Comment:
Need to take img_name from the invalid_names_list(),If it goes above loop, it is referenced before assignment
Anyway to solve this?